dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/bnd.overrides

```diff
@@ -15,7 +15,8 @@ Export-Package: \
   org.apache.cxf.transport.https,\
   org.apache.cxf.transport.servlet,\
   org.apache.cxf.transports.http;version=3.2.1,\
-  org.apache.cxf.transports.http.configuration
+  org.apache.cxf.transports.http.configuration,\
+  org.apache.cxf.transport.stream;version=1.0.2
```

Comment:
don't need this package anymore, right? `org.apache.cxf.transport.stream;version=1.0.2`